personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -22,6 +22,41 @@ function PersonalityTestXBlockStudent(runtime, element) {
         });
         return buff;
     }
+    function addQuestion (question, list, count, studentAnswers) {
+        var questionInList = document.createElement('li');
+        var spanQuestion = document.createElement('div');
+        spanQuestion.className = 'question-div';
+
+        var select = document.createElement('select');
+        var opt = document.createElement('option');
+        opt.text = '';
+        opt.value = '';
+        select.add(opt);
+        var studentAnswer = '';
```

Comment:
this is extremely confusing:

You initialize  `studentAnswer` (without s) to `''`.

Then, just below, you compare `studentAnswers` (with an s) to `''`.

Shouldn’t one of them be an array or something?